### ISSUE TITLE
SBCL Still Fails to Load in Alpine 3.19

### ISSUE THREAD

**Issue by oaguy1** [ish-app/ish#2325]
SBCL (Steele Bank Common Lisp) hasn’t loaded and in fact crashed the app in Alpine 3.14. Now in Alpine 3.19 it still won’t load, but at least this time the app no longer crashes. Happy to help debug this as I very much want a Common Lisp development environment on my phone. Screenshots of the error output below.

![IMG_5235](https://github.com/ish-app/ish/assets/2355614/b59ef097-7593-4c4a-9e67-0855a75ab19b)

![IMG_5239](https://github.com/ish-app/ish/assets/2355614/a0433afa-8777-46b4-83f0-8dbb556328de)

**Comment by saagarjha** [comment]
Do you happen to have a copy of the crash you were seeing before?

**Comment by oaguy1** [comment]
No, I do not. It was different in that it would actually freeze the app and
make the app itself crash. I filed a few crash reports with “sbcl” in them.

**Comment by oaguy1** [comment]
![IMG_5248](https://github.com/ish-app/ish/assets/2355614/758c07b3-baee-472b-b655-35d30503d6d9)

This is what I see

**Comment by saagarjha** [comment]
`clc`. Huh, I guess we don't implement that?

**Comment by oaguy1** [comment]
[This instruction?](https://www.felixcloutier.com/x86/clc) I could be wrong, but this looks like a pretty standard instruction. Happy to help add it if it’s not already in place.

**Comment by saagarjha** [comment]
Yes, but when I run it on my machine it dies even earlier and more mysteriously. So I'm not entirely sure if that would be what we want here.

**Comment by onion108** [comment]
Same issue here with exactly same address when crashing
